pieteikumaforma.html

```diff
@@ -0,0 +1,291 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<head>
+<meta http-equiv="Content-Language" content="lv">
+<meta content="text/html; charset=windows-1257" http-equiv="Content-Type">
+<title>Mana pieteikuma forma</title>
+<meta name="keywords" content="v�rds, uzv�rds, aizrau�an�s, prasmes, zin��anas">
+<meta name="description" content="M�jas darbs Nr2. Mana pieteikuma forma">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<meta name="ROBOTS" content="INDEX,FOLLOW">
+<meta name="Distribution" content="Global">
+<script language="JavaScript" type="text/javascript">
+   WebFontConfig = {
+     google: { families: [ 'Andika::latin-ext',  'Roboto+Slab:400,700:latin-ext', 'Roboto:400,900,500:latin-ext' ] }
+   };
+   (function() {
+     var wf = document.createElement('script');
+     wf.src = ('https:' == document.location.protocol ? 'https' : 'http') +
+       '://ajax.googleapis.com/ajax/libs/webfont/1/webfont.js';
+     wf.type = 'text/javascript';
+     wf.async = 'true';
+     var s = document.getElementsByTagName('script')[0];
+     s.parentNode.insertBefore(wf, s);
+   })(); </script>
+<style type="text/css">
+span.h1 {
+	font-family: Roboto;
+	font-size: 1.5em;  /* 24px; */
+	font-weight: 900;
+	color: #660033;
+	text-decoration: none;
+	vertical-align: baseline;
+	line-height: 150%;
+	word-spacing: 0;
+	text-indent: 0;
+	text-align: center;
+	margin: 2 0;
+	; letter-spacing: 1;
+	text-shadow: 6px 6px 6px #aaa;
+}
+
+span.h2 {
+	font-family: Laila;
+	font-size: 1.125em;   /* 18px;  */ 
+	font-weight: 700;
+	color: #CC0000;
+	text-align: center;
+	text-indent: 0;
+	line-height: 100%;
+	word-spacing: 0;
+	margin: 0;
+}
+
+span.h3 {
+	font-family: Andika;
+	font-size: 1em;
+	font-weight: 400;
+	color: #660033;
+	vertical-align: baseline;
+	letter-spacing: 0;
+	text-indent: 20;
+	line-height: 100%;
+	margin-left: 0;
+	margin-top: 0;
+	margin-bottom: 0;
+}
+
+
+a:active {
+	color: #990033;
+	letter-spacing:.05em;
+	text-decoration: none;
+}
+
+a:hover {
+	letter-spacing:.05em;
+	color: #CC0000;
+	text-decoration: none;
+}
+
+a:link {
+	color: #990033;
+	text-decoration: none;
+}
+
+a:visited {
+	color: #990033;
+	text-decoration: none;
+}
+
+	</style>
+
+
+	</head>
+	<div class="header">
+	<p align="center">
+	<span class="h1">

```

Comment:
You can use heading tags, instead of creating spans with classes
h1-h6 https://www.geeksforgeeks.org/html-heading/#:~:text=An%20HTML%20heading%20tag%20is,level%20and%20H6%20the%20least.